specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ManagedEnvironments.json

```diff
@@ -501,24 +828,167 @@
           "x-ms-secret": true
         }
       }
-    }
-  },
-  "securityDefinitions": {
-    "azure_auth": {
-      "type": "oauth2",
-      "description": "Azure Active Directory OAuth2 Flow",
-      "flow": "implicit",
-      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
-      "scopes": {
-        "user_impersonation": "impersonate your user account"
+    },
+    "Certificate": {
+      "description": "Certificate used for Custom Domain bindings of Container Apps in a Managed Environment",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning state of the Certificate.",
+          "enum": [
+            "Succeeded",
```

Comment:
Do we need this if it is a synchronous API call?

Reply:
Removed it since we won't have a PUT API for the entire collection